src/core/Distribution.pm

```diff
@@ -57,7 +112,7 @@ class Distribution::Resources does Associative {
     }
 
     method AT-KEY($key) {
-        CompUnit::RepositoryRegistry.repository-for-spec($.repo).resource($.dist-id, $key)
+        CompUnit::RepositoryRegistry.repository-for-spec($.repo).resource($.dist-id, "resources/$key")
```

Comment:
Looks like it would be incompatible with dists that were installed on a rakudo version that did not include this change?